bundles/org.eclipse.ui.workbench/Eclipse UI/org/eclipse/ui/internal/messages.properties

```diff
@@ -568,6 +568,23 @@ CheckedTreeSelectionDialog_nothing_available=No entries available.
 CheckedTreeSelectionDialog_select_all=Select &All
 CheckedTreeSelectionDialog_deselect_all=&Deselect All
 
+WindowsDefenderConfigurator_statusCheck=Windows Defender Exclusion Check
+WindowsDefenderConfigurator_exclusionCheckMessage=Microsoft's Windows Defender is active and could significantly decrease the application's start-up and overall performance.\n\
+Select how this installation should be handled by Windows Defender:
+WindowsDefenderConfigurator_exclusionInformation=If Windows Defender is active and scans this application it can significantly slow down its start-up and overall performance.\n\
+To prevent a decrease in performance {0} can exclude itself and all files opened from real-time scanning by Windows Defender.\n\
+Be aware that in general adding exclusions could affect this computer's security and adding them requires Administrator privileges.
```

Comment:
i am unsure if "Administrator" has to be with uppercase A or lovercase a. How do native Windows dialogs spell this?